tools/tests/test_data_builder.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from data_builder import title_to_name_date
+
+tests = [
+    (
+        'Title here | May 30th',
+        ('Title here', '2020-05-30', 'May 30th')
+    ),
+    (
+        'Title here',
+        ('Title here', '', '')
+    ),
+]
+
+
+@pytest.mark.parametrize("input,expected", tests)
+def test_title_to_name_date(input, expected):
+    assert title_to_name_date(input) == expected
+
+
+def test_handle_missing_name(capsys):
+    title_missing_name = '| May 30th'
+    title_to_name_date(title_missing_name)
+
+    captured = capsys.readouterr()
+    assert "Failed name parse: missing name for" in str(captured) 
+
+
+@pytest.mark.skip(reason="failing test, need to handle this case")
```

Comment:
Does this issue exist in the data? If it does - then I think you shouldn't skip it. Should you?

Reply:
This issue does not exist in the current data but I included this just in case we wanted to update the validation to guard against it. Happy to take a stab at that after merging this if it's helpful